addons/website_sale/views/templates.xml

```diff
@@ -2114,8 +2114,8 @@
                                         </div>
                                     </td>
                                 </t>
-                                <td class='td-qty'>
-                                    <div t-esc="line.product_uom_qty" />
+                                <td class='text-center td-qty'>
```

Comment:
The task does not talk about changing the position of the text

Reply:
Yeah true, I forgot to tell Pierre, but I thought that was a mistake so I integrated this change. I know that it should be in be in another task but since it was so small I didn't know what to do

Reply:
It's ok in this task but we should at least ask Pierre :)

Reply:
Yeah ofc u right I just forgot